express-with-nodemon-browsersync/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "express-with-nodemon-browsersync",
+  "version": "1.0.0",
```

Comment:
This is the demo package.json for this recipe. Since it's the first ("stable") version of this recipe, I used 1.0.0. If after this PR we improve it (feature or fix) we'll use semver to increment it.